Improvements and clarifications (#54)
AMWA-TV/ms-05-02: docs/Framework.md

```typescript
// Length method result
interface NcMethodResultLength: NcMethodResult {

>     attribute NcUint32?    value; // Length result value. Can be null if the underlying sequence is null

Reviewer: Should we be a bit more explicit here perhaps?
```suggestion
    attribute NcUint32?    value; // Sequence length result value. MUST be null if the sequence is null
```
